contrib/riscv_update/README.md

```diff
@@ -0,0 +1,57 @@
+# How to update RISCV tables.
+
```

Comment:
please put the link to RISV git repo (upstream) here.

Reply:
Sorry, I can not understand, what should I do?

Reply:
i mean put the URL here, not just the commit ID.